python_src/src/lamp_py/postgres/seed_metadata.py

```diff
@@ -94,10 +96,8 @@ def run() -> None:
     db_manager = DatabaseManager(parsed_args.verbose)
 
     if parsed_args.clear_static:
-        SqlBase.metadata.drop_all(db_manager.engine)
-        drop_alembic_sql = "DROP TABLE IF EXISTS alembic_version;"
-        db_manager.execute(sa.text(drop_alembic_sql))
-        run_alembic_migration("performance_manager")
+        alembic_downgrade_to_base("performance_manager")
+        alembic_upgrade_to_head("performance_manager")
```

Comment:
use alembic to drop all tables from RDS, this is required to correctly move triggers from tables in proper order